app/editor/src/features/admin/topics/TopicList.tsx

```diff
@@ -94,10 +94,13 @@ export const TopicList: React.FC = () => {
         <Col flex="2 1 0">
           <GridTable
             columns={columns}
-            manualPageSize
             isLoading={!!requests.length}
             data={items}
-            paging={{ manualPagination: true, showPaging: false }}
+            paging={{
+              manualPagination: true,
+              type: 'hide',
```

Comment:
The `type` option isn't amazing, but it resolves having different properties make the paging footer show up or hide.  This way it makes sense.